lib/scrabble_word.rb

```diff
@@ -0,0 +1,48 @@
+require "pry"
+
+class ScrabbleWord
+
+  attr_accessor(:word)
```

Comment:
rubyist in general tend to avoid extra parens where not necessary. But if it helps you conceptualize arguments feel free to continute....